### DISCUSSION TITLE
Magick.NET-Q8-AnyCPU, Version=7.10.1.0 to Version=7.22.0.0

### DISCUSSION THREAD

**Discussion by kartikpatts** [dlemstra/Magick.NET#764] [General]
Hello,

i have updated my DLL from Magick.NET-Q8-AnyCPU, Version=**7.10.1.0** to Version=**7.22.0.0**
im not able to rewrite as per new

1. DrawableStrokeColor drawableStrokeColor = new DrawableStrokeColor(Color.Red);
Argument 1: cannot convert from 'System.Drawing.Color' to 'ImageMagick.IMagickColor<byte>'

2. DrawableRectangle drawableRectangle = new DrawableRectangle(myRectangle);
'DrawableRectangle.DrawableRectangle(double, double, double, double)'

how to rewrite as per new 7.22 dll.

**Comment by t246246** [comment]
I had got the same problem, 1st one is easy:

            Color c = Color.Red;
            DrawableStrokeColor drawableStrokeColor = new DrawableStrokeColor(new MagickColor(c.R, c.G, c.B, c.A));

2nd one is probably

           DrawableRectangle drawableRectangle = new DrawableRectangle(myRectangle.Left, myRectangle.Top, myRectangle.Right, myRectangle.Bottom);

I did not try it.

**Comment by dlemstra** [comment]
The constructors for those classes no longer accept `System.Drawing` classes because all the `System.Drawing` logic has moved to the new `Magick.NET.SystemDrawing` package. Both suggestion from @t246246 are correct but you could also use `MagickColors.Red` in the first one instead.

p.s. Always nice to see you helping the ImageMagick community @t246246.
